src/lints/static_unsafe_added.ron

```diff
@@ -0,0 +1,51 @@
+SemverQuery(
+    id: "static_unsafe_added",
+    human_readable_name: "pub static became unsafe",
+    description: "a extern static became unsafe to call.",
+    required_update: Major,  
+    lint_level: Deny,  
+    reference_link: Some("https://doc.rust-lang.org/book/ch19-01-unsafe-rust.html#accessing-or-modifying-a-mutable-static-variable"),  
```

Comment:
I'm a bit confused by this reference link. It seems to be about _mutable_ statics, whereas this lint doesn't check mutability at all.

Is there a more appropriate reference document we can link to?

Reply:
do you think this is a good one: https://doc.rust-lang.org/reference/items/static-items.html